articles/cognitive-services/form-recognizer/concept-business-cards.md

```diff
@@ -25,8 +25,8 @@ The Business Card API extracts key fields from business cards and returns them i
 
 ### Fields Extracted: 
 * Contact Names 
-* First Name 
-* Last Name 
+  * First Names
```

Comment:
```suggestion
  * First names
```

Reply:
Is there reason why we want to lowercase only some of the fields but not all?  I assume we want to be consistent.